### PR TITLE
Switch docs to MicrosoftDocs/CommunityToolkit submodule

### PR DESCRIPTION
**Pull request by Sergio0694**
This PR removes the local copy of the docs and switches to a git submodule.
This eliminates the need to manually keep docs up to date and to have multiple copies around.

### CODE REVIEW

File: .gitmodules [hunk 1]
```diff
@@ -0,0 +1,3 @@
+[submodule "CommunityToolkit"]
+	path = CommunityToolkit
```
Comment (michael-hawker) [inline .gitmodules:2]: Maybe call this `CommunityToolkitDocs` or just `docs_submodule` or something? Not sure if there's a good convention, but would be good to clarify that there's docs in here in the path, though I know the interesting subfolder is also just called `docs`.

File: samples/MvvmSampleUwp/MvvmSampleUwp.csproj [hunk 2]
```diff
@@ -388,55 +388,53 @@
     </Page>
```
Comment (michael-hawker) [inline samples/MvvmSampleUwp/MvvmSampleUwp.csproj:388]: Isn't there a Xamarin head in main, so that needs updating too?

**Review by michael-hawker — COMMENTED** [review]
You should also update the readme with a note how to clone and pull submodules required for building locally?

File: .gitmodules (hunk 1 above)
Comment (Sergio0694) [reply .gitmodules]: Renamed to "msdocs-communitytoolkit". Used lowerCamelCase for consistency with other folder names, I saw ImageSharp is doing the same for their "shared-infrastructure" submodule as well. Plus this name is clearer than before, thanks! 🙂

File: samples/MvvmSampleUwp/MvvmSampleUwp.csproj (hunk 2 above)
Comment (Sergio0694) [reply samples/MvvmSampleUwp/MvvmSampleUwp.csproj:388]: Done in https://github.com/CommunityToolkit/MVVM-Samples/pull/89/commits/07a13d5c424212a78dd490674d92e05c60ceba8f 🙂

**Pull request merged**